packages/api/schema/stryker-core.json

```diff
@@ -290,6 +290,11 @@
       "$ref": "#/definitions/dashboardOptions",
       "default": {}
     },
+    "enableConsoleEmojis": {
```

Comment:
I just now realize that we _do have console options_ here: https://github.com/stryker-mutator/stryker-js/blob/4e5600731cb6c62dfd64cf9b109b0521f342e391/packages/api/schema/stryker-core.json#L61. Would you mind adding it there under the name `allowEmojis` (default false).